src/components/WebView.tsx

```
@@ -1,94 +1,94 @@
-        title="Form UI"
-        name={JSON.stringify(transformData(formData) ?? {})}
-      ></iframe>
-    </Layout>
-  );
+	return (
+		<Layout
+			_heading={{
+				heading: 'Complete Application',
+			}}
+			getBodyHeight={(height) =>
+				(iframeRef.current!.style.height = `${height}px`)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)